vistautils/parameters.py

```diff
@@ -1026,8 +1026,11 @@ def path_list_from_file(
         self, param: str, *, log_name=None, resolve_relative_to: Optional[Path] = None
```

Comment:
mypy fails to catch this (probably because of the behavior Jacob mentions). I'm not sure Python offers a clean way around this, the type is something like
```suggestion
        self, param: Union[str, Sequence[str]], *, log_name=None, resolve_relative_to: Optional[Path] = None
```
and then...